### DISCUSSION TITLE
Work with v1 and v2 on the same dev computer

### DISCUSSION THREAD

**Discussion by KertDawg** [quasarframework/quasar#10380] [General - Components / Directives / etc] [answered]
Hello.  I'm using Mac OS 11 for development of a Quasar v1 app.  I'd like to experiment with Quasar v2 on a new project.  I have been unable to find any information about running v1 and v2 CLI on the same computer.  I would think the solution may be the same for Windows or Unix, so I'm interested in experience with any OS.

Short of working from a VM, what is a good way to have v1 and v2 CLI on the same computer?

**Comment by pdanpdan** [comment, marked as answer]
The club is the same for both versions

**Reply by ontwikkelfabriek** [reply]
@KertDawg 

For Quasar v1 apps, add ` --branch v1` to the end of the create command for creating a new Qv1 app:
https://v1.quasar.dev/quasar-cli/installation#introduction

Probably a good idea to install `nvm`, to be able to switch between node versions:
https://github.com/nvm-sh/nvm
